go.mod

```diff
@@ -7,48 +7,49 @@ require (
 	github.com/armon/go-metrics v0.4.1
 	github.com/cenkalti/backoff v2.2.1+incompatible
 	github.com/deepmap/oapi-codegen v1.11.0
-	github.com/docker/docker v20.10.21+incompatible
-	github.com/envoyproxy/go-control-plane v0.10.2-0.20220325020618-49ff273808a1
+	github.com/docker/docker v23.0.1+incompatible
+	github.com/envoyproxy/go-control-plane v0.10.3
 	github.com/go-logr/logr v1.2.3
 	github.com/golang/mock v1.6.0
-	github.com/golangci/golangci-lint v1.50.1
+	github.com/golangci/golangci-lint v1.51.2
 	github.com/google/uuid v1.3.0
-	github.com/hashicorp/consul-server-connection-manager v0.1.1-0.20221115222743-242744ae73fb
-	github.com/hashicorp/consul/api v1.17.0
-	github.com/hashicorp/consul/proto-public v0.1.0
-	github.com/hashicorp/consul/sdk v0.13.0
+	github.com/hashicorp/consul-server-connection-manager v0.1.2
+	github.com/hashicorp/consul/api v1.19.1
+	github.com/hashicorp/consul/proto-public v0.3.0
+	github.com/hashicorp/consul/sdk v0.13.1
 	github.com/hashicorp/go-changelog v0.0.0-20221013053416-ba40b3a8c7ff
-	github.com/hashicorp/go-hclog v1.3.1
+	github.com/hashicorp/go-hclog v1.4.0
 	github.com/hashicorp/go-multierror v1.1.1
-	github.com/hashicorp/vault/api v1.8.2
-	github.com/hashicorp/vault/sdk v0.6.1
+	github.com/hashicorp/vault/api v1.9.0
+	github.com/hashicorp/vault/sdk v0.8.1
 	github.com/kr/text v0.2.0
 	github.com/mitchellh/cli v1.1.5
 	github.com/mitchellh/mapstructure v1.5.0
 	github.com/prometheus/client_golang v1.14.0
-	github.com/stretchr/testify v1.8.1
-	github.com/vladimirvivien/gexe v0.1.1
+	github.com/stretchr/testify v1.8.2
+	github.com/vladimirvivien/gexe v0.2.0
 	golang.org/x/exp v0.0.0-20220722155223-a9213eeb770e
-	golang.org/x/net v0.2.0
+	golang.org/x/net v0.7.0
```

Comment:
Though this bump fixes some CVEs inside of `golang.org/x/net` we aren't susceptible to them since we don't actually interact with users with our controller, and the only place where we're handling inbound connections anywhere requires being an authorized gateway deployment with a corresponding TLS client certificate -- so if someone was able to craft such requests to our controller, they're already well beyond the security assumptions that we've made here.